docs/code-howtos/testing.md

```diff
@@ -140,6 +140,16 @@ docker run -e MYSQL_ROOT_PASSWORD=root -e MYSQL_DATABASE=jabref -p 3800:3307 mys
 
 Set the environment variable `DBMS` to `mysql`.
 
+## Fetchers in tests
+
+Fetcher tests could be done by executing the Gradle task `fetcherTest`. This could be done by typing the following in the command line:
+
+```shell
+./gradlew fetcherTest
+```
+
+Or, if one is using IntelliJ, by double-clicking `fetcherTest` in the Gradle sidebar; this could be found through the following path: `JabRef > Tasks > other`.
```

Comment:
```suggestion
Alternatively, if one is using IntelliJ, this can also be done by double-clicking the `fetcherTest` task under the `other` group  in the Gradle Tool window (`JabRef > Tasks > other > fetcherTest`).
```